redhat-director-scripts/docker/build_rhosp16_containers.sh

```diff
@@ -9,13 +9,14 @@ echo -e "\tYou can use following commands:"
 echo -e "\n\t- podman login docker.io\n\t- podman login registry.redhat.io\n"
 
 
-if [ $# -lt 1 ];then
-   echo "Script takes exacyly 1 argument"
-   echo -e "./build_container.sh <tvault_version>"
+if [ $# -lt 2 ];then
+   echo "Script takes exactly 2 arguments"
+   echo -e "./build_container.sh <tvault_version> <PUSH_PKG_CNT>"
    exit 1
 fi
 
 tvault_version=$1
+PUSH_PKG_CNT=$2
```

Comment:
@rajneeshkapoor For what purpose we introduced this variable?

Reply:
It's a provision at Jenkins job level to restrict/allow RHOSP16 (_& others as well; for now kolla_) containers upload to respective registry. If this variable set to no or anything other than "yes", code flow will create containers (_& packages too_), but will not upload them, thereby preventing accidental overwriting.

Reply:
Okay.